packages/core/src/Asset/util/assetId.ts

```diff
@@ -10,6 +10,15 @@ export const assetNameFromAssetId = (assetId: AssetId): AssetName => AssetName(a
 export const createAssetId = (scriptHash: CSL.ScriptHash, assetName: CSL.AssetName): AssetId =>
   AssetId(util.bytesToHex(scriptHash.to_bytes()) + util.bytesToHex(assetName.name()).toString());
 
+/**
+ * @returns {AssetId} concatenated policy id and asset name
+ */
+export const assetIdFromPolicyAndName = (policyId: PolicyId, assetName: AssetName): AssetId =>
+  createAssetId(
```

Comment:
Can't we do just `policyId.toString() + assetName.toString()`? Looking at the test for this it seems to be the case